contracts/src/core/EigenDABlobVerifier.sol

```diff
@@ -0,0 +1,302 @@
+     * @param blobHeader The blob header to verify
+     * @param blobVerificationProof The blob verification proof to verify the blob against
+     * @param additionalQuorumNumbersRequired The additional required quorum numbers 
+     */
+    function verifyBlobV1(
+        BlobHeader calldata blobHeader,
+        BlobVerificationProof calldata blobVerificationProof,
+        bytes calldata additionalQuorumNumbersRequired 
+    ) external view {
+        EigenDABlobVerificationUtils._verifyBlobV1ForQuorums(
+            eigenDAThresholdRegistry,
+            eigenDABatchMetadataStorage, 
+            blobHeader, 
+            blobVerificationProof, 
+            bytes.concat(quorumNumbersRequired(), additionalQuorumNumbersRequired)
+        );
+    }
+
+    /**
+     * @notice Verifies a batch of blobs for the required quorums
+     * @param blobHeaders The blob headers to verify
+     * @param blobVerificationProofs The blob verification proofs to verify the blobs against
+     */
+    function verifyBlobsV1(
+        BlobHeader[] calldata blobHeaders,
+        BlobVerificationProof[] calldata blobVerificationProofs
+    ) external view {
+        EigenDABlobVerificationUtils._verifyBlobsV1ForQuorums(
+            eigenDAThresholdRegistry,
+            eigenDABatchMetadataStorage, 
+            blobHeaders, 
+            blobVerificationProofs, 
+            quorumNumbersRequired()
+        );
+    }
+
+    /**
+     * @notice Verifies a batch of blobs for the required quorums and additional quorums
+     * @param blobHeaders The blob headers to verify
+     * @param blobVerificationProofs The blob verification proofs to verify the blobs against
+     * @param additionalQuorumNumbersRequired The additional required quorum numbers
+     */
+    function verifyBlobsV1(
+        BlobHeader[] calldata blobHeaders,
+        BlobVerificationProof[] calldata blobVerificationProofs,
+        bytes calldata additionalQuorumNumbersRequired
+    ) external view {
+        EigenDABlobVerificationUtils._verifyBlobsV1ForQuorums(
+            eigenDAThresholdRegistry,
+            eigenDABatchMetadataStorage, 
+            blobHeaders, 
+            blobVerificationProofs, 
+            bytes.concat(quorumNumbersRequired(), additionalQuorumNumbersRequired)
+        );
+    }
+
+    ///////////////////////// V2 ///////////////////////////////
+
+    /**
+     * @notice Verifies a blob for the base required quorums and the default security thresholds
+     * @param batchHeader The batch header of the blob
+     * @param blobVerificationProof The blob verification proof for the blob
+     * @param nonSignerStakesAndSignature The nonSignerStakesAndSignature for the blob
+     */
+    function verifyBlobV2(
+        BatchHeaderV2 calldata batchHeader,
+        BlobVerificationProofV2 calldata blobVerificationProof,
+        NonSignerStakesAndSignature calldata nonSignerStakesAndSignature
```

Comment:
we talked about this during the call, but would be nice to have a simple API `verifyBlobFromSignedBatchV2(SignedBatch calldata signedBatch)` that would call `_getNonSignerStakesAndSignature`, when doing eth_calls